pandas/io/parsers.py

```diff
@@ -2358,7 +2361,11 @@ def _set(x):
             if is_integer(x):
                 noconvert_columns.add(x)
             else:
-                noconvert_columns.add(self.columns.index(x))
+                # pandas\io\parsers.py:2366: error: Unsupported right
+                # operand type for in ("Optional[List[int]")  [index]
```

Comment:
you need to assert that the result values is not None (assign it to a new variable first).

Reply:
Thanks very much. That helped me a lot.